elftools.gemspec

```diff
@@ -18,6 +18,10 @@ Gem::Specification.new do |s|
   s.files       = Dir['lib/**/*.rb'] + %w[README.md]
   s.homepage    = 'https://github.com/david942j/rbelftools'
   s.license     = 'MIT'
+
+  s.metadata['rubygems_mfa_required'] = 'true'
+  s.metadata['changelog_uri'] = "#{s.homepage}/releases/tag/v#{s.version}",
```

Comment:
need to remove ',' at the end of this line

Reply:
Damn. So sorry for the added work, this was copied from a different contribution I made somewhere else.